.travis.yml

```diff
@@ -1,45 +1,70 @@
-language: c
+language: python
 sudo: false
-services:
-  - postgresql
-env:
-  - TRAVIS_PYTHON_VERSION=2.7
-os:
-  - linux
-  #- osx
-
-# Setup anaconda
-before_install:
+
+cache:
+  pip: true
+  directories:
+  - $HOME/.cache/pip
+  #- $HOME/miniconda3
+
+python:
+    - 2.7
+#    - 3.4
+#os:
+#  - linux
+#  #- osx
+
+addons:
+  apt:
+    packages:
+    - samtools
+
+install:
+  # install planemo for Galaxy tool testing
+  - pip install flake8 planemo
```

Comment:
It doesn't matter, you can also use conda. Important is just that you install planemo with pip.